client/src/featureform/register.py

```diff
@@ -72,6 +72,12 @@
 from .search import search
 from .enums import FileFormat
 
+REGISTER_RESOURCES_WARNING = (
```

Comment:
Can we add a link to the docs for the class API?

Reply:
So ... technically, there's no doc that fully explains it. It's more use-by-example in other docs (e.g. https://docs.featureform.com/quickstart-docker#apply).

So I add a new doc for this?

Reply:
Could link it to this page https://docs.featureform.com/getting-started/defining-features-labels-and-training-sets